### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by nekcoj**
out of bounds checks added

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
       </list>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_13" default="false" project-jdk-name="13.0.1" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_13" default="false" project-jdk-name="13" project-jdk-type="JavaSDK">
```
Comment (lem-n) [inline .idea/misc.xml]: Should be kept as 13.0.1

File: src/main/java/org/app/menu/menus/SearchMenu.java [hunk 2]
```diff
@@ -44,7 +44,7 @@ public void handle() {
             System.out.print("Enter 0 to return or an index to list songs> ");
 
             var index = app.getIntInput();
-            if (index == 0) return;
+            if (index == 0 || index > results.size()) return;
```
Comment (lem-n) [inline src/main/java/org/app/menu/menus/SearchMenu.java]: Maybe add a re-enter prompt?

File: src/main/java/org/app/menu/menus/RemoveMenu.java [hunk 3]
```diff
@@ -33,6 +33,7 @@ public void handle() {
         System.out.print("Enter [index] to remove or press [0] to return to main menu> ");
 
         var removeIndex = app.getIntInput();
+        if(removeIndex <= 0 || removeIndex > results.size()) return;
```
Comment (lem-n) [inline src/main/java/org/app/menu/menus/RemoveMenu.java]: Maybe add a re-enter prompt?

File: src/main/java/org/app/menu/menus/SearchMenu.java (hunk 2 above)
Comment (nekcoj) [reply src/main/java/org/app/menu/menus/SearchMenu.java]: should be fixed now if i understood you correctly.

File: src/main/java/org/app/menu/menus/RemoveMenu.java (hunk 3 above)
Comment (nekcoj) [reply src/main/java/org/app/menu/menus/RemoveMenu.java]: should be fixed now if i understood you correctly.

**Pull request merged**